force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -159,6 +159,15 @@ public inherited sharing class RD2_RecurringDonation {
         return this;
     }
 
+    /**
+    * @description Returns True if the Next Payment Date has changed on an Elevate connected RD
+    * @return Boolean True if the Next Payment Date has changed on an Elevate connected RD
+    */
+    public Boolean needsNextPaymentScheduleCheck() {
+        return changedFields.contains(npe03__Recurring_Donation__c.npe03__Next_Payment_Date__c)
+            && isElevateRecord();
```

Comment:
Is there a reason we aren't clearing the next payment amount for non-elevate RD?

Reply:
Since `isNewOpportunityCandidate` won't happen for Elevate RDs, I needed a different way to clear them. I'm unsure if this will work though, still need to do E2E testing.